.env.example

```diff
@@ -0,0 +1,12 @@
+# NOTE: Using a .env file is optional. The below values are defauled in
+# docker-compose.yml as well, and `docker compose up` works without a .env file.
+
+# The port at which VAST can be accessed. Use
+# `vast -e :42000 <command>` to interact with VAST.
+VAST_PORT=42000
+
+# The Git revision of VAST to use
+# when building VAST images from source. Run `git rev-parse HEAD` to get the current
+# revision, or use the special tag `latest` to refer to the latest upstream
+# revision.
```

Comment:
Can you reflow this and the other comments in this file to 80 columns?

Reply:
Latest commit fixes this. Thanks for pointing this.

Reply:
I also turned on IDE preferences for this. Hopefully it will help in further implementations.